### DISCUSSION TITLE
Trim Command Still Not Work on Brother Innovis NS 1750D

### DISCUSSION THREAD

**Discussion by VikkiInWonderland** [inkstitch/inkstitch#1234] [Q&A]
Hello I've read else where of someone else having problem with their brother embroidery machine not recognizing the trim command on jump stitch.  Such is the case on my Brother Innovis NS 1750D anyone else have problems with the trim command, and is there a solution around it? Thx.

**Comment by tyrosinase** [comment]
That may have been me; my PE800 doesn't trim outside of color changes, but that's a PE800 thing and not an Ink/Stitch thing.

**Reply by VikkiInWonderland** [reply]
tysm! I was told that if you export in DST format, in inkstitch, your brother WOULD recognize the trim commands. I have yet to try this, but do you find this to be true?☻☺♥

**Reply by lexelby** [reply]
I'm afraid that machine doesn't have the ability to trim jump stitches.  It can only trim the thread at the end of a color, and then you have to hold the thread when it starts up again.  Sorry to disappoint!

**Comment by wwderw** [comment]
IF your machine is actually able to recognize trims, there is a "quirk" with Brother machines.  Even if there is a trim function within a color block, but the location of the trim command is within a certain distance of that with the starting point of the next object, it will still do a jump stitch.

DST files do tend to be more reliable with honoring trim functions, but there are still a couple of settings on the Brother machine that you have to make sure that you look at.  This is assuming your machine recognizes trims AND it is also able to read DST files.  Not all single needle machines do.

Really only have a handful of options depending on the capabilities of your machine:

1.  Where you want the trims, create color changes.  This will force the machine to do a trim.  Downside to this is that you will have to babysit the machine in order to start it back up.

2.  Redigitize the design for the trim and start point of the next object are further apart.  Atleast further apart to override Brother's programming on the machine.

3.  Swap to the DST format and see how that fairs.

Again, this does depend on what your machine is able to do as to what option you have available to you.

It really isn't a digitize program's issue, but a machine "issue".

**Comment by tyrosinase** [comment]
On Thu, Jun 10, 2021, at 8:23 PM, wwderw wrote:
> It really isn't a digitize program's issue, but a machine "issue".


Yeah, Brothers tend to be "smart" in some really annoying ways sometimes: sometimes it will decide it needs to do a proper lockdown (small back-and-forth, not just stitch the first few stitches slowly before jumping to full speed) after a halt, if the next stitch is a long enough jump. Normally it's fine but when you're running the machine threadless to punch card-embroidery paper you don't want those extraneous holes.

(Admittedly, I'm sure that's not a use-case that the firmware folks considered. Have I voided my warranty yet? Undoubtedly.)
